### PR TITLE
[CSL-2389] Adds option to consume library as a bundle

### PR DESCRIPTION
**Pull request by Mudaafi**
Users may now consume this library as a bundle via two ways:

#### Via NPM
`npm i '@constructor-io/constructorio-ui-autocomplete'`
```
import CioAutocomplete from '@constructor-io/constructorio-ui-autocomplete/constructorio-ui-autocomplete-bundled';

CioAutocomplete({
  selector: '#autocomplete-container',
  includeCSS: true, // Include the default CSS styles. Defaults to true.
  apiKey: '',
  onSubmit: (submitEvent) => console.dir(submitEvent),
  // ... additional arguments
});
```

#### Via Script Tags (only locally for now)
`<script type="module" src="./constructorio-ui-autocomplete-bundled.js"></script>`
```
window.CioAutocomplete({
  selector: '#autocomplete-container',
  includeCSS: true, // Include the default CSS styles. Defaults to true.
  apiKey: '',
  onSubmit: (submitEvent) => console.dir(submitEvent),
  // ... additional arguments
});
```

### CODE REVIEW

**Review by esezen — APPROVED** [review]
This is looking good. Let's get one more review before we merge

File: README.md [hunk 1]
```diff
@@ -115,6 +117,22 @@ function YourComponent() {
 }
 ```
 
+### Using the Javascript Bundle
+
+This is a framework agnostic method that can be used in any JavaScript project. The `CioAutocomplete` function provides a simple interface to inject an entire Autocomplete UI into the provided `selector`.
+In addition to [Autcomplete component props](https://constructor-io.github.io/constructorio-ui-autocomplete/?path=/docs/autocomplete-component--docs), this function also accepts `selector` and `includeCSS`.
+
+```js
+import CioAutocomplete from '@constructor-io/constructorio-ui-autocomplete/constructorio-ui-autocomplete-bundled';
```
[the remark this answers is not in the document]
Comment (Mudaafi) [reply README.md:126]: Sounds good

File: README.md [hunk 2]
```diff
@@ -115,6 +117,22 @@ function YourComponent() {
 }
 ```
 
+### Using the Javascript Bundle
+
+This is a framework agnostic method that can be used in any JavaScript project. The `CioAutocomplete` function provides a simple interface to inject an entire Autocomplete UI into the provided `selector`.
+In addition to [Autcomplete component props](https://constructor-io.github.io/constructorio-ui-autocomplete/?path=/docs/autocomplete-component--docs), this function also accepts `selector` and `includeCSS`.
```
[the remark this answers is not in the document]
Comment (Mudaafi) [reply README.md:123]: ooh nice eye, will fix

**Review by stanlp1 — APPROVED** [review]
LGTM!

**Review by jjl014 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**